src/CVT.cc

```diff
@@ -481,6 +481,22 @@ void CVT::send( Session* session ){
   }
 
 
+  // Apply repeating watermark if that's been specified
+  Watermark* watermark = session->watermark;
+  if( watermark && watermark->isSet() && (watermark->getMinCVT() > 0)){
+  
+    if( (complete_image.width >= watermark->getMinCVT()) && (complete_image.height >= watermark->getMinCVT())){
```

Comment:
What if `complete_image.height` will not be provided? So the client wants to fetch image only be specifying width of the image but still wants to see watermark.